annotations/src/main/resources/revisions/EffectState/6.json

```diff
@@ -0,0 +1 @@
+{version:6,fields:[{name:color,type:arc.graphics.Color,size:-1},{name:data,type:java.lang.Object,size:-1},{name:effect,type:mindustry.entities.Effect,size:-1},{name:lifetime,type:float,size:4},{name:offsetX,type:float,size:4},{name:offsetY,type:float,size:4},{name:parent,type:mindustry.gen.Posc,size:-1},{name:rotation,type:float,size:4},{name:time,type:float,size:4},{name:x,type:float,size:4},{name:y,type:float,size:4}]}
```

Comment:
You shouldn't commit this file.

Reply:
Sorry; I just added the 2 lines and this file showed up after building Mindustry. I have no idea why.

Reply:
Same happens to me, I just ignore it.